iaso/api/bulk_create_users.py

```diff
@@ -95,15 +98,18 @@ def create(self, request, *args, **kwargs):
             for row in reader:
                 org_units_list = []
                 if i > 0:
-                    try:
-                        validators.validate_email(row[csv_indexes.index("email")])
-                    except ValidationError:
-                        raise serializers.ValidationError(
-                            {
-                                "error": "Operation aborted. Invalid Email at row : {0}. Fix the error and try "
-                                "again.".format(i)
-                            }
-                        )
+                    email_address = True if row[csv_indexes.index("email")] else None
```

Comment:
Pourquoi :

```python
email_address = True if row[csv_indexes.index("email")] else None
    if email_address:
```

Et pas juste : 
```python
if row[csv_indexes.index("email")] :
```
Ou un truc du genre : 

```python 
if csv_indexes.index("email") in row :
```

Je ne sais pas si la clé existe dans row toujours (avec une valeur vide) ou si la clé n'existe pas ... ? Mais tu n'a pas besoin de créer une variable `email_address` que tu n'utilise nulle part ailleur si je ne me trompe pas

Reply:
email n'est pas necessairement dans la ligne, il n'est que dans la premiere. 

 File "/opt/app/iaso/api/bulk_create_users.py", line 101, in create
    if row[csv_indexes.index("email")]:
ValueError: 'email' is not in list